packages/braid-design-system/src/lib/components/Toggle/Toggle.tsx

```diff
@@ -46,11 +50,24 @@ export const Toggle = forwardRef<HTMLInputElement, ToggleProps>(
       align = 'left',
       togglePosition,
       size = 'standard',
+      bleedY = false,
       data,
       ...restProps
     },
     forwardedRef,
   ) => {
+    if (process.env.NODE_ENV !== 'production') {
+      if (bleedY === undefined) {
+        // eslint-disable-next-line no-console
+        console.warn(
+          dedent`
+          Please specify bleed with the "bleedY" prop.
+          Bleed will be the default in a future version.
+          To maintain the current appearance, set "bleedY" to "false"`,
```

Comment:
Feel like swapping the lines and rewording a bit makes it a bit clearer as to why you need to specify an explicit value.

```suggestion
          The default value of "bleedY" will change to "true" in a future version.
          Please specify an explicit value for the "bleedY" prop.
          To maintain the current behaviour, set "bleedY" to "false".`,
```

Reply:
Mostly agree, but I like having the action item first, as the user may not know about the prop. 'Appearance' seems a more specific than behaviour to me. What do you think of:

```
Please specify an explicit value for the "bleedY" prop.
The default value of "bleedY" will change to "true" in a future version.
To maintain the current appearance, set "bleedY" to "false".
```

Reply:
👍